lib/internal/util.js

```diff
@@ -393,6 +393,18 @@ function convertToValidSignal(signal) {
   throw new ERR_UNKNOWN_SIGNAL(signal);
 }
 
+function convertProcessSignalToExitCode(signalCode) {
+  validateString(signalCode, 'signalCode');
+
+  const signalNumber = signals[signalCode];
+  if (signalNumber === undefined) {
```

Comment:
We can use validateOneOf here. e.g: https://github.com/nodejs/node/blob/main/lib/util.js#L149